lib/SPIRV/OCLUtil.cpp

```diff
@@ -143,10 +143,21 @@ unsigned getOCLVersion(Module *M) {
   NamedMDNode *NamedMD = M->getNamedMetadata(kSPIR2MD::OCLVer);
   if (!NamedMD)
     return 0;
-  assert (NamedMD->getNumOperands() == 1 && "Invalid SPIR");
-  MDNode *MD = NamedMD->getOperand(0);
-  unsigned Major = getMDOperandAsInt(MD, 0);
-  unsigned Minor = getMDOperandAsInt(MD, 1);
+  assert (NamedMD->getNumOperands() > 0 && "Invalid SPIR");
```

Comment:
We have a pass TransOCLMD to transform SPIR 1.2/2.0 metadata to SPIR-V friendly metadata. opencl extensions and optional features are handled there. currently there is an assert that they only have one operand. I can fix that.

We don't use spir version or opencl.compiler.options, therefore no need to handle them.

In the LLVM/SPIR-V translation, this function is only used by TransOCLMD to get the correct ocl version. Later passes will use SPIR-V friendly metadata.